database/revert/V20170114_2116__createAttendeeRSVP.revert.sql

```diff
@@ -1 +1,5 @@
-DROP TABLE `attendee_rsvps`;
\ No newline at end of file
+DROP TABLE `attendee_rsvps`;
+DELETE FROM `mailing_lists` WHERE `name`='attendees';
```

Comment:
It's the revert for the `CreateAttendeeRSVP` migration and iirc the revert should reverse the changes that the migration makes completely.